File: Logging.Client/LogEntity.cs
using System.Collections.Generic;

namespace Logging.Client
{
    
    public class LogEntity
    {
        public string Title { get; set; }

        public string Message { get; set; }

        public LogLevel Level { get; set; }

        public long Time { get; set; }

        public string Source { get; set; }

        public int Thread { get; set; }

        public Dictionary<string, string> Tags { get; set; }
    }

   
}

File: client/java/logging.client/src/main/java/logging/client/LogEntity.java
package logging.client;

import java.util.Map;

public class LogEntity extends ILogEntity {

//       public LogEntity() {Type = 1; }

       public String Title;

       public String Message;

       public int Level;

       public long Time;

       public String Source;

       public int Thread;

       public Map<String, String> Tags;
	
}
